subprojects/docs/src/docs/release/notes.md

```diff
@@ -88,6 +88,20 @@ The JAR file is now self-attributing and you don't need to consider whether you
 
 Gradle's [Kotlin DSL](userguide/kotlin_dsl.html) provides an enhanced editing experience in supported IDEs compared to the traditional Groovy DSL — auto-completion, smart content assist, quick access to documentation, navigation to source, and context-aware refactoring.
 
+#### Faster first use
+
+Gradle generates a Gradle API Kotlin DSL Extensions JAR before compiling the first `.gradle.kts` script when the [Gradle User Home](userguide/directory_layout.html#dir:gradle_user_home) is empty.
+This happens on every developer machine or CI agent, often with a cold [Gradle daemon](userguide/gradle_daemon.html).
```

Comment:
The "often with a cold daemon" statement seems misleading. This is not about the daemon being cold, but about the gradle user home not having stuff.

Reply:
It adds up. 99% of the time you'll have both a cold daemon and empty user home, making this even slower.